Detectors/EMCAL/calibration/src/EMCALPedestalHelper.cxx

```diff
@@ -180,17 +180,18 @@ std::vector<char> EMCALPedestalHelper::createInstructionString(const int runNum)
     } // iDTC
   }   // iSM
 
-  //   std::vector<char> output;
-  std::string instructionString(fout.str());
-  //   std::fill(instructionString.begin(), instructionString.end(), std::back_inserter(output));
+  if (runNum > 0) {
+    fout << 0xFFFFFFFF << std::endl;
+  }
+
+  const std::string instructionString(fout.str());
   std::vector<char> output(instructionString.begin(), instructionString.end());
   return output;
 }
 
-void EMCALPedestalHelper::dumpInstructions(const std::string_view filename, const std::vector<char> data, int mRun)
+void EMCALPedestalHelper::dumpInstructions(const std::string_view filename, const gsl::span<char>& data)
```

Comment:
OK, but doesn't really matter for a span to be passed as reference, as the span itself can be considered as a kind of a reference which wraps around arrays.